middleman-core/lib/middleman-core/application.rb

```diff
@@ -75,7 +75,11 @@ def self.root_path
 
     # Which server name should be used
     # @return [NilClass, String]
-    config.define_setting :host, nil, 'The preview host name'
+    config.define_setting :server_name, nil, 'The server name of preview server'
```

Comment:
I renamed host to server_name. I think this is a breaking change though the old setting was more or less usesless from my point of view.